apis/metal3.io/v1alpha1/baremetalhost_validation.go

```diff
@@ -17,11 +17,19 @@ var log = logf.Log.WithName("baremetalhost-validation")
 func (host *BareMetalHost) validateHost() []error {
 	log.Info("validate create", "name", host.Name)
 	var errs []error
-
-	if err := validateRAID(host.Spec.RAID); err != nil {
-		errs = append(errs, err)
+	var bmcAccess bmc.AccessDetails
+
+	if host.Spec.BMC.Address != "" {
+		var err error
+		bmcAccess, err = bmc.NewAccessDetails(host.Spec.BMC.Address, host.Spec.BMC.DisableCertificateVerification)
+		if err != nil {
+			errs = append(errs, err)
+		}
 	}
 
+	errs1 := validateBMCAccess(host.Spec, bmcAccess)
+	errs = append(errs, errs1...)
```

Comment:
Can't this be written in one line `errs = append(errs, validateBMCAccess(host.Spec, bmcAccess)...)`?